app/src/main/res/values/strings.xml

```diff
@@ -324,23 +324,21 @@
     <string name="empty_message_course_certificates">No certificates available for this course</string>
     <string name="empty_message_dates">No upcoming dates right now</string>
 
-    <string name="settings_helpdesk">Help</string>
     <string name="helpdesk_topic_technical_question">Technical question</string>
-    <string name="helpdesk_issue_hint">Issue</string>
+    <string name="hint_issue">Issue</string>
     <string name="helpdesk_info_text">Please describe your technical problem using the textboxes below. Information about your mobile device and your e-mail address will be automatically submitted along with your issue report.</string>
-    <string name="helpdesk_title_hint">Title</string>
-    <string name="helpdesk_topic_hint">Topic</string>
-    <string name="helpdesk_mail_hint">Email</string>
```

Comment:
what happened to the mail hint?

Reply:
Reused Ressource from Line 148
<string name="hint_email">Email</string>

Reply:
But then it is still included in the german resources

Reply:
true, deleted